lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+class Menu
+  attr_reader :items
+
```

Comment:
Did you consider including the option to initialize a menu with a collection of items so as to avoid having to add them all individually?